controllers/member.go

```diff
@@ -16,7 +16,6 @@ package controllers
 
 import (
 	"encoding/json"
-
```

Comment:
No need to remove this. You can set your Go style like this: 

![image](https://user-images.githubusercontent.com/42638489/119666288-d5095680-be67-11eb-9701-df31fa754dcf.png)